1-utils.c

```diff
@@ -0,0 +1,56 @@
+#include "shell.h"
+
+/**
+ * _strncmp - Function that compare two strings
+ * @str1: first string
+ * @str2: second string
+ * @n: amount of bytes to compare
+ *
+ * Return: 0 if second string is found in first string.
+ */
+int _strncmp(const char *str1, const char *str2, size_t n)
+{
+	while (n > 0)
+	{
+		if (*str1 != *str2)
+			return (*str1 - *str2);
+		if ((*str1 == '\0') || (*str2 == '\0'))
+			break;
+		str1++;
+		str2++;
+		n--;
+	}
+
+	return 0;
+}
+
+/**
+ * _strcat_delim - concatenate two strings with the delimiter "/"
+ * @dest: destination of the concatenated string
+ * @str1: first string
+ * @str2: second string
+ *
+ * Return: retrun concatenated string
+ */
+char *_strcat_delim(char *dest, char *str1, char *str2)
```

Comment:
This function name can be more descriptive e.g. `construct_full_path()`, especially since it deviates from the normal `strcat` and specifically concatenates with delimiters